### ISSUE TITLE
docs: Issue in page SELECT * FROM MUTATION_FRAGMENTS() Statement - page seems out of place

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#17324]
I would like to report an issue on page https://opensource.docs.scylladb.com/branch-5.4/operating-scylla/admin-tools/select-from-mutation-fragments

### Problem

The header looks (literally) out of line in the tool index
![image](https://github.com/scylladb/scylladb/assets/170200/62efb0ad-fb19-4ecd-833a-0975303114d2)

###  Suggest a fix
Renaming the page to something like "Reading mutations data" would look better.

**Comment by tzach** [comment]
Another alternative is moving this page to the KB (and rename)

**Comment by tzach** [comment]
Its more of a CQL virtual table than a "tool", but I'm all for mimizing updates, so ignore lets my move comment.

**Comment by denesb** [comment]
> Its more of a CQL virtual table than a "tool"

Yes, kind-of (initially I even wanted to make it just a virtual table), but note that we also have the virtual tables in this list above. Yes, it is kind-of a mess, everything that can be used for diagnostics was thrown into this list.
